src/components/UNSTABLE__Shell/PopoverFooter/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file Popover footer.
+ * @copyright IBM Security 2020
+ */
+
+import { node } from 'prop-types';
+import React from 'react';
+
+import { appendComponentNamespace } from '../../../globals/namespace';
+import namespace from '../Popover';
+
+function PopoverFooter({ children, ...other }) {
+  return (
+    <footer
+      className={appendComponentNamespace(namespace, 'footer')}
```

Comment:
```suggestion

const footerNamespace = appendComponentNamespace(namespace, 'footer');

function PopoverFooter({ children, ...other }) {
  return (
    <footer
      className={footerNamespace}
```

Could we do this? Likely may need to export the namespace for tests 🤔

Reply:
Here, I wanted to keep everything as private as possible until we find the necessity to expose it as an export.

I can imagine we may eventually need these for unit tests, but I'd rather do that once we identify that we do, if that makes sense. Otherwise, we run the risk of exposing it to consumers (I'm not sure how real a concern this is), but if so, it becomes harder to remove down the line.